Mathlib/MeasureTheory/Measure/Haar/OfBasis.lean

```diff
@@ -293,4 +293,13 @@ protected def measurableEquiv : EuclideanSpace ℝ ι ≃ᵐ (ι → ℝ) where
 theorem coe_measurableEquiv : ⇑(EuclideanSpace.measurableEquiv ι) = WithLp.equiv 2 _ := rfl
 #align euclidean_space.coe_measurable_equiv EuclideanSpace.coe_measurableEquiv
 
+theorem coe_measurableEquiv_symm :
+    ⇑(EuclideanSpace.measurableEquiv ι).symm = (WithLp.equiv 2 _).symm := rfl
+
+theorem coe_continuousLinearEquiv :
+    ⇑(EuclideanSpace.measurableEquiv ι) = (EuclideanSpace.equiv ι ℝ) := rfl
+
+theorem coe_continuousLinearEquiv_symm :
+    ⇑(EuclideanSpace.measurableEquiv ι).symm = (EuclideanSpace.equiv ι ℝ).symm := rfl
```

Comment:
These lemmas don't actually coerce to `ContinuousLinearEquiv`, do they?

Reply:
I guess the name is not optimal but the idea is that `EuclideanSpace.equiv` is a the continuous linear equiv from which `EuclideanSpace.measurableEquiv` is constructed...

Reply:
I'd recommend just dropping these lemmas; they follow by transitivity of the two lemmas above, and the "simplifying" lemma that says `EuclideanSpace.equiv` is just `WithLp.equiv`